client/galaxy/scripts/components/admin/Jobs.vue

```diff
@@ -0,0 +1,371 @@
+        },
+        onRefresh() {
+            this.update();
+        },
+        onUpdateJobLock() {
+            axios
+                .get(`${getAppRoot()}admin/jobs_control?job_lock=${this.jobLock}`)
+                .then(response => {
+                    this.jobLockDisplay = response.data.job_lock;
+                })
+                .catch(error => {
+                    console.error(error);
+                });
+        },
+        onStopJobs() {
+            this.update();
+            this.selectedStopJobIds = [];
+            this.stopMessage = "";
+        },
+        showRowDetails(row, index, e) {
+            if (e.target.nodeName != "A") {
+                row._showDetails = !row._showDetails;
+            }
+        },
+        translateState(state) {
+            const translateDict = {
+                ok: "success",
+                error: "danger",
+                new: "primary",
+                queued: "secondary",
+                running: "info",
+                upload: "dark"
+            };
+            return translateDict[state] || "primary";
+        },
+        computeItems(items) {
+            return items.map(job => {
+                return {
+                    ...job,
+                    _showDetails: false,
+                    _cellVariants: { state: this.translateState(job.state) }
+                };
+            });
+        },
+        computeFields(fields) {
+            let f = Array.from(fields).slice(0);
+            if (this.showCommandLine) {
+                f.splice(6, 0, {
+                    key: "command_line",
+                    tdClass: ["code", "break-word"]
+                });
+            }
+            return f;
+        },
+        toggleAll(checked) {
+            this.selectedStopJobIds = checked
+                ? this.jobsItemsModel.reduce((acc, j) => [...acc, j["job_info"]["id"]], [])
+                : [];
+        }
+    },
+    computed: {
+        jobsItemsComputed() {
+            return this.computeItems(this.jobsItems);
+        },
+        recentJobsItemsComputed() {
+            return this.computeItems(this.recentJobsItems);
+        },
+        jobsFieldsComputed() {
+            let f = this.jobsFields.slice(0);
+            f.splice(0, 0, { key: "selected", label: "", variant: "info" });
+            f.splice(2, 0, { key: "update_time", label: "Last Update", sortable: true });
+            return this.computeFields(f);
+        },
+        recentJobsFieldsComputed() {
+            let f = this.jobsFields.slice(0);
+            f.splice(2, 0, { key: "update_time", label: "Finished", sortable: true });
+            return this.computeFields(f);
+        }
+    },
+    created() {
+        this.update();
+    }
+};
+</script>
+
+<style>
+/* Can not be scoped because of command line tdClass */
+.code {
+    background: black;
+    color: white;
+    padding: 1em;
+}
+.break-word {
+    white-space: pre-wrap;
+    word-break: break-word;
+}
+</style>
```

Comment:
Can we remove the custom styles here, particularly those which specify colors and spacing? There are some alternative code-related style tags available: https://getbootstrap.com/docs/4.0/content/code/.